FRs #1055/1073: Refactor Provider Identification, Host Provider, Invoice Issuer (#1139)
FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec: specification/datasets/cost_and_usage/columns/providername.md

Reviewer: I feel like we should make some declaration here that Provider has been replaced by Service Provider.  If this markdown is not the appropriate place for that, then where?  The changelog only?  We could, but that doesn't feel sufficient.

Author: Good point. I like your suggested changes.

Reviewer 2: ```suggestion
Provider
```

Why are we updating a deprecated column in the version that we're deprecating the column?

Author: We were trying to align with the other name change. It is going to happen there anyway.

Reviewer: Per [this note](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/issues/1380#issuecomment-3391120404) from @Matt-Cowsert, it was agreed in Oct 10 TF3 to revert the changes to ProviderName.  Let's do that per this PR; that is a material change that is, in my opinion, outside the bounds of consistency review.

Reviewer: @shawnalpay will take an action to resolve this.

> # Provider Name - DEPRECATED

Reviewer: ```suggestion
# Provider - DEPRECATED
```

> Provider Name is the name of the entity that makes the [*resources*](#glossary:resource) or [*services*](#glossary:service) available for purchase. It is commonly used for cost analysis and reporting scenarios.

Reviewer: ```suggestion
Provider is the name of the entity that makes the [*resources*](#glossary:resource) or [*services*](#glossary:service) available for purchase. It is commonly used for cost analysis and reporting scenarios.
```

* ProviderName MUST conform to [StringHandling](#stringhandling) requirements.
* ProviderName MUST NOT be null.

> See [Appendix: Entity Identification Examples](#entityidentification) section for examples of [Service Provider Name](#serviceprovidername), [Host Provider Name](#hostprovidername) and [Invoice Issuer Name](#invoiceissuername) values that can be used for various purchasing scenarios.

Reviewer: @rileyjenk 
The appendix link no longer worked, so in the latest commit, I replaced it with the new one, and I scrubbed reference to Provider Name because that's no longer mentioned in that appendix.  Let me know if you think that's okay, and if so, you can resolve this thread.

Author: Thanks!

> See [Appendix: Entity Identification Examples](#entityidentification) section for examples of [Service Provider Name](#serviceprovidername), [Host Provider Name](#hostprovidername) and [Invoice Issuer Name](#invoiceissuername) values that can be used for various purchasing scenarios.

Reviewer 3: ```suggestion
See [Appendix: Participating Entity Identification Examples](#participatingentityidentificationexamples) section for examples of [Service Provider Name](#serviceprovidername), [Host Provider Name](#hostprovidername) and [Invoice Issuer Name](#invoiceissuername) values across various use case scenarios.
```

> See [Appendix: Participating Entity Identification Examples](#participatingentityidentificationexamples) section for examples of [Service Provider Name](#serviceprovidername), [Host Provider Name](#hostprovidername) and [Invoice Issuer Name](#invoiceissuername) values across various use case scenarios.

Reviewer: Revise this verbiage to make clear that this is deprecated, and leave a link to a previous version in GitHub.

Alternatively, link to the glossary, and then that glossary entry points to the external link.

In either case, make clear that this an external link, not within the PDF/HTML.

Reviewer: ```suggestion
See [Appendix: Participating Entity Identification Examples](#participatingentityidentificationexamples) section for examples of [Service Provider Name](#serviceprovidername), [Host Provider Name](#hostprovidername) and [Invoice Issuer Name](#invoiceissuername) values across various use case scenarios.  For entity identification examples that include the deprecated Provider column, please see [here](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/blob/7296e3146fb931ee32b6496fa6e4200e7f4384f7/specification/appendix/origination_of_cost_data.md) (external GitHub link to FOCUS v1.2).
```

## Introduced (version)

0.5

## Deprecated (version)

> 1.3

Reviewer: ```suggestion
1.3 Replaced by [ServiceProviderName](#serviceprovidername)
```

## Introduced (version)

0.5

> ## Deprecated (version)

Reviewer 2: Title (column display name at the top on line one) should have - DEPRECATED? that would match with Publisher?
